itest/fee_estimation_test.go

```diff
@@ -17,13 +17,15 @@ import (
 func testFeeEstimation(t *harnessTest) {
 	var (
 		// Make a ladder of UTXO values so use order is deterministic.
-		anchorAmounts = []int64{10000, 9990, 9980, 9970}
+		anchorAmounts = []int64{1_000_000, 999_990, 999_980, 99_970}
```

Comment:
We had to bump the values here in order to not run into the 20% `FundPsbt` fee check anymore. Also the last output was shrinked in order to return the expected errors which are fee calculation related (output must fail for excessive fee)